t/unicodable.t

```diff
@@ -217,6 +217,14 @@ $t.test(‘last basic query, just in case’,
         /^ <me($t)>‘, U+2A34 MULTIPLICATION SIGN IN LEFT HALF CIRCLE [Sm] (⨴)’ $/,
         /^ <me($t)>‘, U+1D227 GREEK INSTRUMENTAL NOTATION SYMBOL-17 [So] (𝈧)’ $/);
 
+# https://github.com/perl6/whateverable/issues/234
+$t.test(｢\U lookup by code｣,
+        “{$t.bot-nick}: \\U0010ffff”,
+        /^ <me($t)>‘U+10FFFF <illegal> [] (�)’ $/);
```

Comment:
Interesting, are you using rakudo 2018.09 or less? The result is different now: https://gist.github.com/b6d312cb369774379a5c61f5a1d7d080